src/averageAges.js

```diff
@@ -15,11 +15,19 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const manAge = people.filter(person => {
+    return century
+      ? person.sex === 'm'
+       && Math.ceil(person.died / 100) === century
+      : person.sex === 'm';
+  })
```

Comment:
```suggestion
  const manAge = people.filter(person => century
    ? person.sex === 'm'
      && Math.ceil(person.died / 100) === century
    : person.sex === 'm';
 )
```

в стрелочная функция сразу return делает, можно без фигурных скобок записать

Reply:
Да у меня там просто ошибка была, вот и подумал, что лучше так написать). Сейчас будет исправлено, спасибо))